streams/runners/storm/actions/src/main/java/com/hortonworks/streamline/streams/actions/storm/topology/StormTopologyActionsImpl.java

```diff
@@ -941,4 +940,21 @@ private LogLevelInformation convertLogLevelResponseToLogLevelInformation(LogLeve
         return LogLevelInformation.enabled(LogLevel.valueOf(resp.getTargetLevel()), resp.getTimeoutEpoch());
     }
 
+    //storm.py expects each -c config option to be in json since StormSubmitter eventually uses a json parser for converting it to java object
+    //nimbus.seeds is expected to be a list of strings or List<String> in java. Json representation of that is ["a", "b"]
+    //note that you need double quotes in above json representation and single quotes are not valid
+    //Furthermore shell removes any double quotes from command line arguments when it passes to python.
+    //More on that at http://teaching.idallen.com/cst8207/12w/notes/280_quotes.txt
+    //As a result we need to pass something like [\"a\", \"b\"] to the command line. Without escaping the double quotes python will get arguments as [a,b] and
+    //that will fail the json parsing in storm submitter java code. Escaping of double quote for command line argument is taken care by java
+    private String getNimbusSeedsForCommandLine() {
+        StringBuilder nimbusSeedsJsonString = new StringBuilder("[");
+        if ((nimbusSeeds != null) && !nimbusSeeds.isEmpty()) {
+            String[] seeds = nimbusSeeds.split(",");
+            for (String seed: seeds) {
```

Comment:
This would be safer if we use stream() and Collectors.joining() or `String.join()`. 

Currently it adds "," by end of array, and assuming you tested before then it doesn't break the functionality, but still not beauty. :)

Reply:
@HeartSaVioR Will change stream() and join as you suggested. Comma at the end is valid for json string. I tested it and it works fine. In fact, i did not want that extra code after the loop to remove the last character and hence kept it as is.